src/libs/ajax.js

```diff
@@ -1754,12 +1755,23 @@ const Martha = signal => ({
   },
 
   getSignedUrl: async ({ bucket, object, dataObjectUri }) => {
-    const res = await fetchMartha('getSignedUrlV1',
+    const res = await fetchMartha(
+      'getSignedUrlV1',
       _.mergeAll([jsonBody({ bucket, object, dataObjectUri }), authOpts(), appIdentifier, { signal, method: 'POST' }]))
     return res.json()
   }
 })
 
+const DrsHub = signal => ({
+  getDataObjectMetadata: async (url, fields) => {
+    const res = await fetchDrsHub(
+      '/api/v4/drs/resolve',
```

Comment:
Doesn't this need to still be `'martha_v3'` for non-dev environments?

Reply:
Ah, I was so excited to get this tested that I forgot to add a flag for dev

Reply:
updated!